### PR TITLE
fix : Changed looks of Cart

### LINKED ISSUE #31: Enhancements for Add to cart button
## Description
The `Add to cart` is bit off-place, and it looks bad !

## Expected Behavior
If you take a proper look, the add to cart is bit off-place, specially for mobiles ! 
Also, such a large button is **not a good UI**, for smaller screens.

I can fix it, and also replace the gray button with `a minimal cart icon` which will look much clean !

#### Image 1
image

## Environment:
- Host OS:  Windows 10
- Browser: Brave Browser

### PR DESCRIPTION
**Pull request by tamalCodes**
## Description 🗒

 - I have fixed and worked on #31 , you can check out the demo video !
 - I have updated some versions , else app won't start.
 - Sorry for being late, had some emergency health issues.

## Demo 📹

https://user-images.githubusercontent.com/72851613/187045470-d86bbd01-ff1a-4be6-b56a-a9d402df78a2.mp4

## Important  ⚠

 - Updated to `"react-scripts": "5.0.1",`
 - Installed/Updated Material Icons
 
---

**This PR closes #31**

**Regards,**
**Tamal**

### CODE REVIEW

**Review by kamalbuilds — CHANGES_REQUESTED** [review]
An add word with a space and then the cart icon would look better :thinking:

File: src/components/Home.js [hunk 1]
```diff
@@ -6,42 +6,45 @@ import { addBasket } from '../actions/addAction';
...
+                <a onClick={() => props.addBasket('rust')} className="addToCart cart1" href="#">
+                    <AddShoppingCartIcon className='addToCart_icon' />
+                </a>
             </div>
             <div className="image">
                 <img src={go} alt="go" />
                 <h3>Go</h3>
                 <h3>$35.00</h3>
-                <a onClick={() => props.addBasket('go')} className="addToCart cart2" href="#">Add to Cart</a>
+                <a onClick={() => props.addBasket('go')} className="addToCart cart2" href="#"> <AddShoppingCartIcon className='addToCart_icon' /></a>
```
Comment (kamalbuilds) [inline src/components/Home.js]: <a onClick={() => props.addBasket('go')} className="addToCart cart2" href="#"> Add <AddShoppingCartIcon className='addToCart_icon' /></a>

Like this elsewhere too.

**Review by kamalbuilds — CHANGES_REQUESTED** [review]

**Comment by tamalCodes** [comment]
Will be done ✅

**Comment by kamalbuilds** [comment]
Nice

**Review by kamalbuilds — CHANGES_REQUESTED** [review]
I will say it again , no need to add package-lock.json just copy it from the master branch and replace , inorder words revert those changes they always cause merge conflicts which make it tough to merge your changes as you can see :eye:

**Pull request closed without merging**